simapp/app_v2.go

```diff
@@ -168,6 +168,7 @@ func NewSimApp(
 			depinject.Supply(
 				// supply the application options
 				appOpts,
+				logger,
```

Comment:
It would be great to have a diff in the upgrading.md about that for people going from v0.47 to v0.48 using depinject.